lib/GridItem.jsx

```diff
@@ -40,7 +40,7 @@ type GridItemCallback<Data: GridDragEvent | GridResizeEvent> = (
 ) => void;
 
 type State = {
-  resizing: ?{ width: number, height: number },
+  resizing: ?{ top: number, left: number, width: number, height: number },
```

Comment:
This suggests: can a resize be represented as a combination of a width/height change and a top/left drag?